utils/telemetry/events.go

```diff
@@ -0,0 +1,41 @@
+package telemetry
+
+import (
+	"time"
+)
+
+type event struct {
+	id          string
+	eventType   EventType
+	userID      string
+	time        time.Time
+	executionID string
+	command     string
+	version     string
+	data        []EventData
+}
+
+// EventData outlines a generic structure for tracking events
+type EventData struct {
+	Key   string
+	Value interface{}
+}
+
+// EventType signifies where in the command this event occurred
+type EventType string
+
+// set of supported EventTypes
+const (
+	EventTypeCommandStart EventType = "COMMAND_START"
+	EventTypeCommandEnd   EventType = "COMMAND_END"
```

Comment:
likewise to match `master` ... we use `COMMAND_COMPLETE`, so let's update this value (and `var` name here(